### ISSUE TITLE
Problem publishing slidified file: missing style modification and link to "local" a picture is broken

### ISSUE THREAD

**Issue by tarrade** [ramnathv/slidify#474]
Hi there,

I am stuck and I couldn't find a solution to my problems.

I made some slides with slidify with:
- a logo image to be included in the title slide and located in assets/img
  In YAMLI added:  "logo        : bkg.png"
- an image to be inlcuded in a slide and located in assets/img
  "< img class="center" src="./assets/img/map.png" width=600 >"
- I changed the color of the text and of the background of the tittle slides
  I made a modification in libraries/frameworks//io2012//css/slidify.css
- I am using in YAML: "mode        :  selfcontained" but I am the same issue if I use (standalone instead)
# run locally :

  => all perfect
# publishing with gihub and gh-page

  => all perfect
# use the publish button in RStudio :

  empty page published and no error
# use the command line:

  publish(title = 'My title', 'index.html', host = 'rpubs')

Many issue :
1) the color of the text and the color of the background of the slide is the default one and not the one I changed !

2) the picture included with <  img class="center" s rc="./assets/img/map.png" width=600 > is missing and I couldn't find a way to fix this issue
 The logo picture is present

so it look like:
- ./libraries/frameworks//io2012//css/slidify.css folder is not used/found
- ./assets/img/map.png is not found
  while:
- ./assets/img/bkg.png is used and found

Which command should I put to force slidify to use my local libraries when publishing my slides ?
Where should I put the pictures I want to use in my slides ? Under /assets/img ? but this work only localy and not when I published the slides.

Thanks
Cheers
Fabien

ps : I did set setwd("my_local_dir") than contain all the needed files and directories

```
   My setup :
```

R version 3.2.2 (2015-08-14)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.10.5 (Yosemite)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] plotly_2.0.10   mapproj_1.2-4   maps_3.0.1      dplyr_0.4.3     Rmisc_1.5       plyr_1.8.3      lattice_0.20-33
 [8] ggplot2_1.0.1   markdown_0.7.7  shiny_0.12.2    slidify_0.5    

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.2      viridis_0.3.1    base64enc_0.1-3  bitops_1.0-6     tools_3.2.2      testthat_0.11.0
 [7] digest_0.6.8     jsonlite_0.9.19  memoise_0.2.1    gtable_0.1.2     DBI_0.3.1        parallel_3.2.2  
[13] swirl_2.2.21     yaml_2.1.13      proto_0.3-10     gridExtra_2.0.0  httr_1.0.0       stringr_1.0.0  
[19] knitr_1.11       htmlwidgets_0.5  grid_3.2.2       R6_2.1.1         rmarkdown_0.8.1  reshape2_1.4.1  
[25] magrittr_1.5     whisker_0.3-2    scales_0.3.0     htmltools_0.2.6  MASS_7.3-45      assertthat_0.1  
[31] mime_0.4         xtable_1.8-0     colorspace_1.2-6 httpuv_1.3.3     labeling_0.3     stringi_1.0-1  
[37] lazyeval_0.1.10  RCurl_1.95-4.7   munsell_0.4.2    crayon_1.3.1

**Comment by ramnathv** [comment]
Rpubs only supports the upload of a single file. So currently all js/css assets are served from a Google CDN. As a result, nothing local will get picked up. I implemented a `onefile` branch [here](https://github.com/slidify/onefile), which might do the trick in creating a single file with all assets inlined. It is still an experimental implementation, so might be a bit buggy.

**Comment by tarrade** [comment]
Hi Ramnath,

Thanks a lot for the quick feedback. What surprise me is that the logo file under asset was used.

I did a quick try with your proposal but I guess I am missing one ingredient or a updated package verion.

I downloaded you branch and tried the following:

setwd("/Users/tarrade/Desktop/Cours/Coursera - Developing Data Products/Project/Project2/onefile")
library(devtools)
devtools:::install_github('ramnathv/slidify@onefile')
library(slidifyLibraries)
library(knitr)
slidify("index.Rmd")

here I am using your files only but I get:

"
Error in if (body$content == "") { : 
  missing value where TRUE/FALSE needed"

Thanks a lot
Cheers
Fabien

ps: gh page and github are doing the job but I wanted to give a try to your solution
